src/scripts/h5p-dictation-sentence.js

```diff
@@ -330,9 +330,11 @@ class Sentence {
 
     if (this.buttonPlayNormal) {
       this.buttonPlayNormal.reset();
+      this.buttonPlayNormal.resetAudio();
```

Comment:
I guess calling `resetAudio` could as well be done as part of `Button.reset` in general instead of calling after calling `Button.reset`, couldn't it?